internal/pkg/apply/apply.go

```diff
@@ -68,7 +69,12 @@ func (a *Apply) Do() (Result, error) {
 				fmt.Fprintf(os.Stderr, "failed to update inventory object %v\n", err)
 			}
 		}
-
+		if presence, ok := annotation[constants.Presence]; ok {
+			if presence == constants.EnsureNoExist {
+				// not applying the resource
```

Comment:
how does this get deleted?

Reply:
I updated the prune function to accept a slice of resources. Based on the annotation of each resource, the prune can delete the proper object. This works without a configmap.